flavors/anbox/default.nix

```diff
@@ -0,0 +1,76 @@
+# SPDX-FileCopyrightText: 2021 Samuel Dionne-Riel
+# SPDX-FileCopyrightText: 2021 Daniel Fullmer and robotnix contributors
+# SPDX-License-Identifier: MIT
+
+{ config, pkgs, lib, ... }:
+let
+  inherit (lib)
+    mkDefault
+    mkIf
+    mkMerge
+  ;
+  anboxBranch = "pmanbox";
+  repoDirs = lib.importJSON (./. + "/repo-${anboxBranch}.json");
+in mkIf (config.flavor == "anbox")
+{
+  androidVersion = mkDefault 7;
+
+  # product names start with "anbox_"
+  #  → lunch anbox_arm64-user
+  productNamePrefix = "anbox_";
+
+  buildDateTime = mkDefault 1623041218;
+
+  # anbox uses this by default. If your device supports it, I recommend using variant = "user"
+  #variant = mkDefault "userdebug";
+  variant = mkDefault "user";
+
+  source.dirs = mkMerge ([
+    repoDirs
+    #{
+    #  "xxxtable/recovery" = {
+    #    patches = [
+    #      ./patches/xxxxoid_bootable_recovery/0001-xxxxx.patch
+    #    ];
+    #  };
+    #}
+    {
+      "build" = {
+        # Pre-seed mount points required for bind mounts
+        postPatch = ''
+          mkdir -p $out/blueprint
+          mkdir -p $out/kati
+          mkdir -p $out/soong
+          mkdir -p $out/make
+        '';
```

Comment:
I have vague memories of having to do something similar with another android repo manifest.  Perhaps we could make this automatic--if we are using the bind-mount stuff and we notice one repo would be mounted inside of another one, we patch the outside repo to contain these mount points

Reply:
The workaround where you manually added mount points should be fixed by: https://github.com/danielfullmer/robotnix/pull/104